### ISSUE TITLE
Session Reply SwiftUI Views Redaction

### ISSUE THREAD

**Issue by iramzyy** [getsentry/sentry-cocoa#4333]
### Problem Statement

Hi,

I'm currently trying to use session reply in my app, but I've noticed that if I want to exclude certain views (SwiftUI Views) for privacy reasons, the `redactViewTypes` option is only available for classes (as it's an array of the `AnyClass` type).

Is there a workaround for this?

**Comment by brustolin** [comment]
Hello @iramzyy, thanks for reaching out.

Its not possible right now, we will work on a modifier solution in the coming weeks.

**Comment by iramzyy** [comment]
@brustolin  Is this merged ?  as i'm trying to use the beta version  `8.37.0-beta.1` but i'm getting: 

Value of type 'SwiftUI' has no member 'replayRedact' , My Implementation:

```
import SwiftUI
import SentrySwiftUI

 CustomSwiftUIView()
    .replayRedact()

```

**Comment by iamrita** [comment]
@iramzyy did #4362 fix your issue?

**Comment by brustolin** [comment]
Hello @iramzyy, this is not merged yet. we will release a new beta soon.

**Comment by iamrita** [comment]
when is the new beta being released?

**Comment by brustolin** [comment]
Hello @iamrita, this feature is already available in the release 8.40.0.

[Here](https://docs.sentry.io/platforms/apple/guides/ios/session-replay/customredact/) are some documentation about masking.
